### DISCUSSION TITLE
Instagram

### DISCUSSION THREAD

**Discussion by Ayato-kosaka** [Ayato-kosaka/react-itinerary#315] [General]
Instagram について話し合う

**Comment by Ayato-kosaka** [comment]
2019年のInstagram利用者割合
![image](https://user-images.githubusercontent.com/53715839/201512947-4b82f344-b8fa-4d3c-9fb0-2c680a23c264.jpeg)

**Comment by Ayato-kosaka** [comment]
ロールモデル調査

**Reply by Ayato-kosaka** [reply]
@Takapy1 
旅行の情報系発信アカウント調べてみた
時系列でやってるのはなかった〜

1. **@lonelyplanet** - ロンリープラネット
    - 内容: 世界中の旅行情報やヒントを提供するガイドブックの公式アカウント。
  
2. **@travelandleisure** - トラベル＆レジャー
    - 内容: 世界中の旅行スポットやホテル、レストランのレビューや情報を提供。
  
3. **@cntraveler** - コンデナスト トラベラー
    - 内容: ラグジュアリーな旅行情報や目的地のガイドを提供する旅行雑誌の公式アカウント。
  
4. **@thepointsguy** - ザ・ポインツ・ガイ
    - 内容: 旅行のハックやマイル、ポイントの情報を中心に、旅行者向けの情報を提供。
  
5. **@culturetrip** - カルチャー トリップ
    - 内容: 世界中の文化や食、アートに関する情報やヒントを提供。
  
6. **@tripadvisor** - トリップアドバイザー
    - 内容: 世界中の観光スポットやホテルのレビューと写真をフィーチャー。
  
7. **@atlasobscura** - アトラス・オブスキュラ
    - 内容: 世界中の隠れた場所や未知の観光スポットに関する情報を提供。
  
8. **@passionpassport** - パッション・パスポート
    - 内容: 旅行者やクリエイターとのコラボレーションを通じて、旅行のインスピレーションや情報を提供。
  
9. **@roughguides** - ラフ・ガイド
    - 内容: 世界中の旅行情報やヒントを提供するガイドブックの公式アカウント。
  
10. **@budgettravel** - バジェット トラベル
    - 内容: 予算内での旅行のヒントや情報を提供。

その他、アプリの競合で instagram やってるのないかな〜って見てみたけど、
時系列でやってるのはなかった〜
↓例えばfunliday
https://www.instagram.com/funliday_jp/

**Reply by Ayato-kosaka** [reply]
@Takapy1 

https://www.instagram.com/p/CtaB0Mcsh7H/?igshid=MzRlODBiNWFlZA==

こういう投稿って次の投稿と繋がりが見えるから、
時系列のつながりを表すのに参考になりそう

あとは、

https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ32tWmSTKx0P05RUtIBh9EO883t8hUotTgsQ&usqp=CAU

みたいな感じで、矢印でつながりを表現するとか良さそう。

**Reply by Takapy1** [reply]
@Ayato-kosaka 
ごめん、俺が確認できてないだけや

**Reply by Takapy1** [reply]
この1個目って一枚の画像になってるのかな？
それともわざと次の投稿ではみ出てるように見せるために編集してるのかな

> @Takapy1 
> 
> https://www.instagram.com/p/CtaB0Mcsh7H/?igshid=MzRlODBiNWFlZA==
> 
> こういう投稿って次の投稿と繋がりが見えるから、
> 時系列のつながりを表すのに参考になりそう
> 
> あとは、
> 
> https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ32tWmSTKx0P05RUtIBh9EO883t8hUotTgsQ&usqp=CAU
> 
> みたいな感じで、矢印でつながりを表現するとか良さそう。

**Reply by Ayato-kosaka** [reply]
@Takapy1 

https://www.instagram.com/p/Cu4MQ2_vUdJ/?igshid=MzRlODBiNWFlZA==

モデルプラン乗っけてる投稿あったわ

**Reply by Ayato-kosaka** [reply]
https://www.instagram.com/p/CwE_IeZyMq5/?igshid=MzRlODBiNWFlZA==

これもだ

**Reply by Takapy1** [reply]
@Ayato-kosaka 
おーこれはロールモデルに使えそう！
